doc/cosign_exit_codes.md

```diff
@@ -0,0 +1,8 @@
+# Exit codes for cosign CLI
```

Comment:
This is great but I'm worried that we'll lock ourselves in too soon. Can you add a note that these *may* be useful to script against, but may change in the future? 

Then if we're happy after a few months we can stabilize.

Reply:
Yep, sure thing. I'll add a comment in the code